rosbag2_py/src/rosbag2_py/_info.cpp

```diff
@@ -41,12 +42,30 @@ class Info
     return info_->read_metadata(uri, storage_id);
   }
 
-  void read_metadata_and_output_service_verbose(
-    const std::string & uri,
-    const std::string & storage_id)
+  void print_output(
+    rosbag2_storage::BagMetadata metadata_info)
+  {
+    // Output formatted metadata
+    std::cout << format_bag_meta_data(metadata_info) << std::endl;
+  }
+
+  void print_output_topic_name_only(
+    rosbag2_storage::BagMetadata metadata_info)
```

Comment:
Nitpick: Please use const reference and it could be in one line
```suggestion
  void print_output_topic_name_only(const rosbag2_storage::BagMetadata & metadata_info)
```